EtlPipelineBlobTrigger/course_stats.py

```diff
@@ -160,24 +174,28 @@ def get_stats(self, raw_course_data):
 class Nss:
     """Extracts and transforms the NSS course element"""
 
+    NUM_QUESTIONS = NUM_QUESTIONS = 27
```

Comment:
Im not sure you ment this? Surely `NUM_QUESTIONS = 27` would suffice?

Reply:
:+1: missed that - thanks